Update cloud_cost_management.md (#15582)
DataDog/documentation: content/en/infrastructure/cloud_cost_management.md

## Tag enrichment

Datadog automatically ingests all of your [AWS Resource tags][6] for most of your resources, without you having to manually [activate them][8] as cost allocation tags in your bill. This allows you to easily allocate costs to the dimensions your organization cares about. 

Additionally, Datadog adds out-of-the-box tags to the ingested cost data to help you further break down and allocate your costs. These tags are derived from your [Cost and Usage Report (CUR)][7].

The following out-of-the-box tags are also available for filtering and grouping data:

| Tag                          | Description       |
| ---------------------------- | ----------------- |
| `aws_product`                | The AWS service being billed.|
| `aws_product_family`         | The category for the AWS service being billed (for example, Compute or Storage).|
| `aws_management_account_name`| The AWS management account name associated with the item.|

> | `aws_management_account_id`  | The AWS management account id associated with the item.|

Reviewer: ```suggestion
| `aws_management_account_id`  | The AWS management account ID associated with the item.|
```

| `aws_member_account_name`    | The AWS memmber account name associated with the item.|

> | `aws_member_account_id`      | The AWS member account id associated with the item.|

Reviewer: ```suggestion
| `aws_member_account_id`      | The AWS member account ID associated with the item.|
```

> | `aws_cost_type`              | The type of charge covered by this item (for example, Usage, or Tax).|

Reviewer: ```suggestion
| `aws_cost_type`              | The type of charge covered by this item (for example, Usage, Tax, and so on).|
```

> | `aws_pricing_term`           | Whether the usage is Reserved, Spot, or On Demand.|

Reviewer: ```suggestion
| `aws_pricing_term`           | Whether the usage is Reserved, Spot, or On-Demand.|
```

| `aws_reservation_arn`        | The ARN of the Reserved Instance that the item benefited from.| 
| `aws_savings_plan_arn`       | The ARN of the Savings Plan the item benefited from.|
| `aws_usage_type`             | The usage details of the item (for example, BoxUsage:i3.8xlarge).|
| `aws_operation`              | The operation associated with the item (for example, RunInstances).|
| `aws_region`                 | The region associated with the item.|
| `aws_availability_zone`      | The availability zone associated with the item.|

> | `aws_resource_id`            | The resource id associated with the item.|

Reviewer: ```suggestion
| `aws_resource_id`            | The resource ID associated with the item.|
```

| `aws_instance_type`          | The instance types associated with your items.|
| `aws_instance_family`        | The instance family associated with your item (for example, Storage optimized).|
| `is_aws_ec2_compute`         | Whether the usage is related to EC2 compute.|

> | `is_aws_ec2_compute_on_demand`| Whether the usage is on demand.|

Reviewer: ```suggestion
| `is_aws_ec2_compute_on_demand`| Whether the usage is on-demand.|
```

> | `is_aws_ec2_compute_reservation`| Whether the usage associated with a Reserved Instance.|

Reviewer: ```suggestion
| `is_aws_ec2_compute_reservation`| Whether the usage is associated with a Reserved Instance.|
```

> | `is_aws_ec2_capacity_reservation`| Whether the usage associated with a Capacity Reservation.|

Reviewer: ```suggestion
| `is_aws_ec2_capacity_reservation`| Whether the usage is associated with a Capacity Reservation.|
```
